lib/app/modules/ImportExport/controllers/import_controller.dart

```diff
@@ -111,31 +110,20 @@ class ImportController extends GetxController
   }
 
   void pickImportFile() async {
-    try {
-      final importPath = await FlutterFileDialog.pickFile(
-        params: const OpenFileDialogParams(
-          fileExtensionsFilter: ['json'],
-          mimeTypesFilter: ['application/json'],
-        ),
-      );
-
-      if (importPath == null) {
-        return;
-      }
-
-      final js = json.decode(await File(importPath).readAsString())
-          as Map<String, dynamic>;
-      toImport.value = ImportSelections.fromJson(js);
-    } catch (e) {
-      // unawaited(analytics.logEvent(name: Events.ImportFail, parameters: {
-      //   'reason': e.toString(),
-      // }));
+    var result =
+        await FilePicker.platform.pickFiles(type: FileType.custom, allowedExtensions: ['json', 'dungeonpaper']);
+    if (result == null) {
       Get.rawSnackbar(
-        title: tr.backup.importing.error.title,
-        message: tr.backup.importing.error.message,
+        title: S.current.importFailedTitle,
+        message: S.current.importFailedMessage,
```

Comment:
I recently updated the translation module, now it should be:

```suggestion
        title: tr.backup.importing.error.title,
        message: tr.backup.importing.error.message,
```